draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -382,7 +405,94 @@
       </section> 
     </section>
 
-      
+
+    <section anchor="cdni-metadata-footprint-types" title="CDNI Metadata Footprint Types">
+      <t>The CDNI Footprint Object is defined in section 4.2.2.2 of the CDNI Metadata interface document
+          <xref target="RFC8006"/>. The object is composed of a footprint-type, referring few of the simple data-types
```

Comment:
"is defined in section 4.2.2.2 of the CDNI Metadata interface document <xref target="RFC8006"/>" ==>
"is defined in the CDNI Metadata interface document, see section 4.2.2.2 of <xref target="RFC8006"/>"
This way you get a direct link to the relevant section.